### PR TITLE
flakebox

### PR DESCRIPTION
**Pull request by sectore**
Update `flake.nix` to use `Flakebox` instead of `Fedimint` as input.

Needed to update to latest Rust (see https://github.com/elsirion/webimint-rs/pull/40#issuecomment-1937846664) 

```
cargo version
cargo 1.75.0 (1d8b05cdd 2023-11-20)
```

Notes: 

- `nix.flake` has been created by following [Tutorial: Flakebox in a New Project](https://github.com/rustshop/flakebox/blob/cd7cc8aabdb555b0b01233c727d76257cbd9c130/docs/building-new-project.md) and by adding `wasm` toolchain inspired by [Fedimint's `flake.nix` ](https://github.com/fedimint/fedimint/blob/master/flake.nix) and  [Nostrmint's `flake.nix`](https://github.com/m1sterc001guy/nostrmint/blob/990cb8c642201b25ccd1cd8302153cc5659f0b3d/flake.nix)
- Some extra files are auto-generated by `Flexbox`, mostly in folders `.config/flakebox/` and `misc`, but also a `just` file has been added. Those files can be deleted if not needed.
- Rename app name from `fedimint-leptos-test` to `webimint`

### CODE REVIEW

**Review by elsirion — APPROVED** [review]
I think this is good enough since it unblocks #40. I think we should fix our `justfile` though.

**Comment by elsirion** [comment]
Also big thanks to @dpc for building `falkebox`!

File: justfile [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# THIS FILE IS AUTOGENERATED FROM FLAKEBOX CONFIGURATION
+!include justfile.fedimint.just
```
[the remark this answers is not in the document]
Comment (dpc) [reply justfile:2]: Latest flakebox `master` has a fix for the syntax.

The file gets generated if you call `flakebox install` and only contains any imports if they were passed via `config` key in the `flake.nix`.

[the remark this answers is not in the document]
Comment (dpc) [reply justfile:2]: To sum up: Call `flakebox install` and it should be gone. If it's still is something must be doing: https://github.com/fedimint/fedimint/blob/663dd0350ec0d265d77712c6fa85967a13b8a10b/flake.nix#L135

[the remark this answers is not in the document]
Comment (sectore) [reply justfile:2]: Fixed in #60

**Pull request merged**